src/hooks/__tests__/useFocusStyle.test.tsx

```diff
@@ -0,0 +1,98 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as React from 'react';
+
+import useFocusStyle, { focusAttributeName } from '../useFocusStyle';
+
+const focusClassName = 'focused';
+
+const TestComponent: React.FC = () => {
+  const container = React.useRef<HTMLDivElement | null>(null);
+  useFocusStyle({
+    container,
+    className: focusClassName,
+  });
+
+  return (
+    <div>
+      <button type="button">Outside button</button>
+      <div ref={container}>
+        <button type="button">Button 1</button>
```

Comment:
We could try also add some other type of element that usually dont have focus here, like:
`<div tabIndex="1">Focusable text</div>`
and test it too.

Reply:
Added divs to test